aspnetcore/signalr/hubs/sample/Hubs/StronglyTypedChatHub.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.AspNetCore.SignalR;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace SignalRChat.Hubs
+{
+    public class StronglyTypedChatHub : Hub<IChatClient>
+    {
+        public async Task SendMessage(string user, string message)
+        {
+            await Clients.All.ReceiveMessage(user, message);
+        }
+
+        public Task SendMessageToCaller(string message)
+        {
+            return Clients.Caller.ReceiveMessage(message);
+        }
+
+        public Task SendMessageToGroups(string message)
+        {
+            List<string> groups = new List<string>() { "SignalR Users" };
```

Comment:
Remove the list, this supports passing a single argument. Use Group instead of Groups

Reply:
Ah.  I'll fix that in `ChatHub.cs` too - I just refactored the original Hub.